kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ScanSuite.scala

```diff
@@ -1086,7 +1080,7 @@ class ScanSuite extends AnyFunSuite with TestUtils with ExpressionTestUtils with
       checkResults(
         predicate = nullSafeEquals(col("value"), ofNull(STRING)),
```

Comment:
And maybe switch these args as well (since we already have `col <=> null` above)?

Reply:
Done